client/src/components/AllFeedback/index.js

```diff
@@ -20,21 +20,21 @@ const AllFeedback = ({ student }) => {
 	//Render the page on first load
 	useEffect(()=>{
```

Comment:
Can we add isEditing state (or the function that inverse it currently called noShow) as a dependency to useEffect, So everytime isEditing changes value it will render with the updated feedbacks. This way we wont need to pass updateFeedback as a prop to EditFeedback, and will avoid calling the updateFeedback() again inside the fetch of saveFeedback function in client/src/components/EditFeedback/index.js: line 29 below.

Reply:
I will be changing this entirely.